files/default/sql/ddl/1.3.0__initial_tables.sql

```diff
@@ -1532,6 +1532,7 @@ CREATE TABLE `training_dataset` (
   `hopsfs_training_dataset_id` INT(11) NULL,
   `external_training_dataset_id` INT(11) NULL,
   `training_dataset_type`   INT(11) NOT NULL DEFAULT '0',
+  `seed` INT(11) NULL,
```

Comment:
As you set the Java type to `Long` this field should be `BigInt`

Reply:
fixed